doc/tutorials/01-lennard_jones/CMakeLists.txt

```diff
@@ -1,8 +1,12 @@
-configure_tutorial_target(
-  TARGET tutorial_01
-  DEPENDS
-  01-lennard_jones.ipynb
-  figures/lennard-jones-potential.png
-)
+configure_tutorial_target(TARGET tutorial_01 DEPENDS 01-lennard_jones.ipynb
+                          figures/lennard-jones-potential.png)
 
-nb_export(TARGET tutorial_01 SUFFIX "" FILE "01-lennard_jones.ipynb" HTML_RUN VAR_SUBST)
+nb_export(
+  TARGET
+  tutorial_01
+  SUFFIX
+  ""
+  FILE
+  "01-lennard_jones.ipynb"
+  HTML_RUN
+  VAR_SUBST)
```

Comment:
The majority of the changes looks very good, but in tutorials, the changes look inconsistent: the `configure_tutorial_target` command is nicely condensed while the `nb_export` command is split over too many lines; can't we have it condensed over two lines only? This would be below the 80 chars limit:
```cmake
nb_export(TARGET tutorial_04 SUFFIX "1"
          FILE "04-lattice_boltzmann_part1.ipynb" HTML_RUN)
```

Reply:
all the other tutorials are still affected

Reply:
I think you looked at outdated files.